internal/pkg/aws/ecs/ecs.go

```diff
@@ -333,13 +333,35 @@ func (e *ECS) ActiveClusters(arns ...string) ([]string, error) {
 
 	var active []string
 	for _, cluster := range resp.Clusters {
-		if aws.StringValue(cluster.Status) == clusterStatusActive {
+		if aws.StringValue(cluster.Status) == statusActive {
 			active = append(active, aws.StringValue(cluster.ClusterArn))
 		}
 	}
 
 	return active, nil
+}
 
+// ActiveServices returns the subset of service arns that have an ACTIVE status.
+func (e *ECS) ActiveServices(clusterARN string, serviceARNs ...string) ([]string, error) {
+	resp, err := e.client.DescribeServices(&ecs.DescribeServicesInput{
+		Cluster:  aws.String(clusterARN),
+		Services: aws.StringSlice(serviceARNs),
```

Comment:
What happens if there are no arns specified? Is this an error, or does the DescribeServices call describe all services on the cluster?

Can we add a unit test looking for this behavior?

Reply:
very good call will add